code-studio/package.json

```diff
@@ -22,9 +22,8 @@
     "build-css": "./build-css.js",
     "build-css:min": "./build-css.js --min",
     "build-js": "./build-js.js",
-    "build-js:min": "./build-js.js --min",
```

Comment:
I'm removing the min task because it's no longer supported.  I'll be restoring a min/dist option in a subsequent PR.